specification/portal/resource-manager/readme.md

```diff
@@ -24,18 +24,19 @@ For other options on installation see [Installing AutoRest](https://aka.ms/autor
 
 These are the global settings for the portal.
 
-```yaml
+``` yaml
 openapi-type: arm
-tag: package-2015-11-01-preview
+tag: package-preview-2015-08
```

Comment:
Should be `package-2015-08-01-preview` to be consistent with the other files? Please search and replace any instances of `package-preview-2015-08` - I see a few others in this file.

Reply:
I can change it, but this is how the OpenAPIPortal generated the tags. I'm not sure if this is the new format. I can change it back to the previous if that's what is should be.

Reply:
@NelsonDaniel , @praries880 What is the expected format here.

Reply:
@sergey-shandar can you look into why is the portal generating the tags like this?

Reply:
@jjbfour you will need to update the tag to match the format in the rest of the file

Reply:
@billytrend @vladbarosan any idea why portal produces such tags?